ppr-ui/src/components/common/Attention.vue

```diff
@@ -71,7 +76,7 @@ export default defineComponent({
     const { maxLength } = useInputRules()
 
     const localState = reactive({
-      config: isRoleManufacturer.value ? attentionConfigManufacturer : attentionConfig,
+      config: props.configOverride || isRoleManufacturer.value ? attentionConfigManufacturer : attentionConfig,
```

Comment:
This is just one example but ideally we don't couple our components to specific configurations (if they are configurable). 
Utilizing props or slots is always preferred for reuse. 
Being agile, we end up like this sometimes, all good, just providing a path forward for the component, can always clean up as we go.

Reply:
Looking closely at the config, it is actually a simple content text. The parent should take care of it (based on role) and just pass the right content to the component. It has been done to a few components. And then we won't need configs or overrides.

Reply:
The content may be simple but in my opinion, a single config or resource object is preferable over 5-6 props etc. 
But definitely agree it should be passed to the child component in 1 or the other form.

Reply:
Yea only one resource object could be passed (with all the props) which can be added/removed without changing the component's signature, eg:
```
 :content="{
          title: 'Document ID',
          description: 'Enter the 8-digit Document ID number.',
          sideLabel: 'Document ID'
        }"
```